app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -92,15 +92,15 @@
       </transition>
     </div>
     <div class="right-wrap">
-      <div v-if="isFormatMode" class="enter-to-send--checkbox">
+      <div v-if="isFormatMode" class="keyboard-to-send--checkbox">
```

Comment:
instead of keybpoard, we could try `keypress`

Reply:
Done https://github.com/chatwoot/chatwoot/pull/4401/commits/510e8c7384565badf09661ecf39eb7fbbfaaca66